modules/Hoot/tools/selectBbox.js

```diff
@@ -176,6 +212,10 @@ export default class SelectBbox extends EventEmitter {
         this.maxLonInput.property( 'value', this.maxlon );
         this.minLatInput.property( 'value', this.minlat );
 
+        if ( this.bboxSelectType !== 'boundsHistory' ){
+            this.dropdownContainer.node().value = '';
```

Comment:
Also, to wear my d3 badge on my chest, can we use some d3 here instead? 
```javascript
d3.select(...).attr('value', '')
```

Reply:
🤓 um actually...you would use `.property` instead but i know what you meant 🤓

Reply:
lol